docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -195,48 +207,61 @@ EOF
 After you have created the `Issuer` and updated the values, add it to your cluster using `kubectl`:
 
 ```code
-$ kubectl create namespace teleport
+$ kubectl create namespace <Var name="namespace" />
 $ kubectl label namespace teleport 'pod-security.kubernetes.io/enforce=baseline'
-$ kubectl --namespace teleport create -f aws-issuer.yaml
+$ kubectl --namespace <Var name="namespace" /> create -f aws-issuer.yaml
 ```
 
 </TabItem>
 <TabItem label="AWS Certificate Manager">
 In this step, you will configure Teleport to use AWS Certificate Manager (ACM)
 to provision your Teleport instances with TLS credentials.
 
+<Admonition type="warning" title="Prerequisite: Install and configure the AWS Load Balancer controller">
+  You must either follow the [AWS-maintained documentation on installing the AWS Load Balancer Controller](https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html)
+  or already have a working installation of the AWS LB controller before continuing with these instructions. Failure to do this will result in an unusable Teleport cluster.
+</Admonition>
+
 To use ACM to handle TLS, add annotations to the chart specifying the ACM
-certificate ARN to use and the port it should be served on.
+certificate ARN to use, the port it should be served on and other ALB configuration
+parameters.
 
 Replace
 `arn:aws:acm:us-east-1:1234567890:certificate/12345678-43c7-4dd1-a2f6-c495b91ebece`
 with your actual ACM certificate ARN.
 
-Edit your `values.yaml` file to complete the `annotations.service` field as
-follows:
+Edit your `values.yaml` file to set values as follows:
```

Comment:
This is an awkward flow now because we only show the ACM instructions if people explicitly open that section. I'll try to condense the values down into the section below so we only mention values once.